src/core/mac/mac.hpp

```diff
@@ -739,6 +752,9 @@ class Mac: public InstanceLocator
 
     otMacCounters mCounters;
     uint32_t mKeyIdMode2FrameCounter;
+
+    SuccessRateTracker mCcaFailureRateTracker;
```

Comment:
nit: the naming seems contradictory - type is "SuccessRate" and member is "FailureRate".  I get that `SuccessRateTracker` have methods to provide both, but it seems more natural to call this member "SuccessRate" since "AddSample" takes `true` for success.

Reply:
Good point! Will change it.